src/utils/TypeAssertion.ts

```diff
@@ -1,4 +1,114 @@
 import type { Unknownify } from "../types";
+import type { NetscriptContext } from "../Netscript/APIWrapper";
```

Comment:
I have only one concern, which is: Does this increase our circular dependency load?